cpp/src/cider/exec/nextgen/operators/ColumnToRowNode.cpp

```diff
@@ -104,6 +107,84 @@ class ColumnReader {
     }
   }
 
+  //| BOOL     | INT64      | INT8*                |   C_TYPE*      |  INT64     |
+  //|----------|------------|----------------------|----------------|------------|
+  //| row_null | row_length | elem_null for column | values for row | row_offset |
```

Comment:
Better to provide an example value about these representations, like using `[[12, -7, 25], null, [0, -127, 127, 50], []]` as in the arrow doc.